cmd/chain_info/arbitrum_chain_info.json

```diff
@@ -0,0 +1,357 @@
+{
+  "42161" : {
+    "chain-name" : "arb1",
+    "parent-chain-id" : 1,
+    "chain-parameters": {
+      "persistent" : {
+        "chain": "arb1"
+      },
+      "node": {
+        "forwarding-target": "https://arb1-sequencer.arbitrum.io/rpc",
+        "feed": {
+          "input": {
+            "url" : "wss://arb1.arbitrum.io/feed"
+          }
+        }
+      },
+      "l1": {
+        "rollup": {
```

Comment:
We should move the rollup addresses outside of the CLI config and into their own top-level section like `chain-config`

Reply:
done.